README.md

```diff
@@ -32,6 +32,12 @@ There are also several optional command-line parameters:
  - `-v`: Generate verbose output.
  - `-u pwd`: Provide the user PIN/password.
  - `-o pwd`: Provide the security officer PIN/password.
+ -  `-w` : Wrap mechanism { 
```

Comment:
nit: could we make this more specific, and also tweak the layout to be a bit more consistent:
```md
- `-w cnm: Name of cipher to use for keys being wrapped in key-wrapping tests. One of {
...
```